dashboard/app/models/programming_environment_category.rb

```diff
@@ -9,6 +9,7 @@
 #  color                      :string(255)
 #  created_at                 :datetime         not null
 #  updated_at                 :datetime         not null
+#  position                   :integer
```

Comment:
Should position be allowed to be null? I think we would need to wait to make it not null until its set but figured I would ask now while thinking about it.

Reply:
Probably would be best yes. But I didn't in this PR for the reason you're referring to. I'm planning on setting these positions as a follow-up then I adjust after.